snapcraft.yaml

```diff
@@ -0,0 +1,23 @@
+name: retdec
+version: "8e65c05"
```

Comment:
Shouldn't we use a stable version (e.g. `3.2`, which is the latest stable version of RetDec) instead of a particular commit in the current `master`?

Reply:
I haven't tested it but it should be able to checkout a specific release tag i.e. 3.2

Reply:
Thank you for all the answers. Could I ask you to update the `snapcraft.yml` file so it builds the [v3.2](https://github.com/avast-tl/retdec/releases/tag/v3.2) version, which is the current stable release of RetDec?

Reply:
Hey @s3rvac, sorry have been really busy and had no time do do this yet but I haven't forgotten.

Reply:
That's alright ;-).